.github/renovate.json

```diff
@@ -12,11 +12,6 @@
       "groupName": "npm minor",
       "matchManagers": ["npm"],
       "matchUpdateTypes": ["minor", "patch"]
```

Comment:
Should `npm minor` get the same treatment? I imagine we'd have the same issues there

Reply:
Yeah. I checked the bitwarden/server repo as well and they do not have the npm grouping. Prepare for a lot of renovate prs :D